operator/deploy/csv/apicurio-registry.clusterserviceversion.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+    categories: Streaming & Messaging
+    certified: "false"
+    # DO NOT COMMIT AUTOMATIC MODIFICATIONS TO THIS FILE
+    containerImage: quay.io/apicurio/apicurio-registry-operator:latest-snapshot
+    createdAt: "0000-00-00"
+    description: Deploy and manage Apicurio Registry 3 on Kubernetes.
+    repository: https://github.com/Apicurio/apicurio-registry/tree/main/operator
+    support: Apicurio
+    operators.openshift.io/infrastructure-features: '[]'
+    features.operators.openshift.io/disconnected: "false" # TODO: Support disconnected install
+    features.operators.openshift.io/proxy-aware: "false"
+    features.operators.openshift.io/fips-compliant: "false"
+    features.operators.openshift.io/tls-profiles: "false"
+    features.operators.openshift.io/token-auth-aws: "false"
+    features.operators.openshift.io/token-auth-azure: "false"
+    features.operators.openshift.io/token-auth-gcp: "false"
+    features.operators.openshift.io/cnf: "false"
+    features.operators.openshift.io/cni: "false"
+    features.operators.openshift.io/csi: "false"
+  name: apicurio-registry.v0.0.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+      - description: ApicurioRegistry3 represents an instance of Apicurio Registry version 3.
+        displayName: Apicurio Registry 3
+        kind: ApicurioRegistry3
+        name: apicurioregistries3.registry.apicur.io
+        version: v1
+  description: |
+    ## Apicurio Registry 3
+
+    Apicurio Registry stores and retrieves API designs and event schemas,
+    and gives you control of their evolution.
+
+    **Features**
+    - TODO
```

Comment:
@EricWittmann I'd appreciate some input on what to put here. The original list is:
```
    **Features**
    - Supports: Apache Avro, AsyncAPI, GraphQL, JSON Schema, Kafka Connect Schema, OpenAPI, Protobuf
    - Provides a REST API and web UI to manage the artifacts and artifact meta-data
    - Includes Serializers and Deserializers for Kafka client integration
    - Configurable rules to control schema validity and evolution (compatibility)
    - Storage options: Kafka, PostgreSQL, in-memory
    - Compatible with Confluent and IBM APIs
    - Runs on a lightweight Quarkus platform
    - Includes Maven plugin to integrate with Maven based builds
```